docs/setup-and-installing.md

```diff
@@ -154,7 +158,7 @@ To run a cloud deployment, you need to prepare your server-worker and client-wor
 
 ## Setting up Actor replication
 
-Unreal provides a system called [Actor replication](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors) (Unreal documentation) to make it easy to make a networked game. The SpatialOS Unreal GDK allows you to continue using the native Unreal workflow without changes to your game code. However, you need to do an additional step in order for Actor replication to work with the SpatialOS Unreal GDK.
+Unreal provides a system called [Actor replication](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors) to make it easy to make a networked game. The SpatialOS Unreal GDK allows you to continue using the native Unreal workflow without changes to your game code. However, you need to do an additional step in order for Actor replication to work with the SpatialOS Unreal GDK.
```

Comment:
I thought it might be handy to say that this isn't our documentation, but it might be that it's obvious given the context! (Is that what your reasoning was?)

Reply:
Yeah, that was my reasoning. I knew what it was going to be before I clicked on it.